org.eclipse.xtext.tests/src/org/eclipse/xtext/util/UriUtilTest.java

```diff
@@ -0,0 +1,55 @@
+/*******************************************************************************
+ * Copyright (c) 2015, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.xtext.util;
+
+import java.io.File;
+import org.eclipse.emf.common.util.URI;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+import static org.eclipse.xtext.util.UriUtil.*;
+
+public class UriUtilTest {
+	@Test
+	public void testPrefix() {
+		URI prefix = URI.createURI("file:/foo/");
+		URI uri = URI.createURI("file:/foo/bar");
+		assertTrue(isPrefixOf(prefix, uri));
```

Comment:
I would prefer making the assert statements more explicit by changing `assertTrue(isPrefixOf(prefix, uri))` to `assertTrue(UriUtil.isPrefixOf(prefix, uri))`

while going through the tests, it takes few seconds for you to realize that the method `#isPrefixOf(...)` is statically imported. I am not sure if you feel the same. Please make the change only if it makes sense to you.